### ISSUE TITLE
"The token '&amp;&amp;' is not a valid statement separator in this version."  Windows error

### ISSUE THREAD

**Issue by Schwad** [inspec/inspec#5214]
Affiliated with #5213 

This is an error uncovered when running our test suite against windows machines. We have some `skip_windows!` methods that we are trying to slowly remove in order to have our windows coverage on par with all other platforms. 

We see this in the following places:

- [ ] All three specs in `lib/plugins/inspec-init/test/functional/inspec_init_plugin_test.rb` We expect a normal stdout message but receive an output of:
```
#<struct Train::Extras::CommandResult
 stdout="",
 stderr=
  "#< CLIXML\r\n<Objs Version=\"1.1.0.1\" xmlns=\"http://schemas.microsoft.com/powershell/2004/04\"><S S=\"Error\">At line:1 char:100_x000D__x000A_</S><S S=\"Error\">+ ... d C:/Users/polyn/AppData/Local/Temp/d20200817-7964-1kniov7 &amp;&amp; C:/Ruby ..._x000D__x000A_</S><S S=\"Error\">+                                                                ~~_x000D__x000A_</S><S S=\"Error\">The token '&amp;&amp;' is not a valid statement separator in this version._x000D__x000A_</S><S S=\"Error\">    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException_x000D__x000A_</S><S S=\"Error\">    + FullyQualifiedErrorId : InvalidEndOfLine_x000D__x000A_</S><S S=\"Error\"> _x000D__x000A_</S></Objs>",
 exit_status=1>
```
- [ ] In  `lib/plugins/inspec-init/test/functional/inspec_init_profile_test.rb` we also expect a normal stdout but receive an error of this on all tests:
```
#<struct Train::Extras::CommandResult
 stdout="",
 stderr=
  "#< CLIXML\r\n<Objs Version=\"1.1.0.1\" xmlns=\"http://schemas.microsoft.com/powershell/2004/04\"><S S=\"Error\">At line:1 char:99_x000D__x000A_</S><S S=\"Error\">+ ... cd C:/Users/polyn/AppData/Local/Temp/d20200817-948-1atzkeu &amp;&amp; C:/Ruby ..._x000D__x000A_</S><S S=\"Error\">+                                                                ~~_x000D__x000A_</S><S S=\"Error\">The token '&amp;&amp;' is not a valid statement separator in this version._x000D__x000A_</S><S S=\"Error\">    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException_x000D__x000A_</S><S S=\"Error\">    + FullyQualifiedErrorId : InvalidEndOfLine_x000D__x000A_</S><S S=\"Error\"> _x000D__x000A_</S></Objs>",
 exit_status=1>
```

- [ ] Both specs in `lib\plugins\inspec-plugin-manager-cli\test\functional\help_test.rb` have this same error expecting a stdout but receive:
```
 #<struct Train::Extras::CommandResult
 stdout="",
 stderr=
  "#< CLIXML\r\n<Objs Version=\"1.1.0.1\" xmlns=\"http://schemas.microsoft.com/powershell/2004/04\"><S S=\"Error\">At line:1 char:117_x000D__x000A_</S><S S=\"Error\">+ ... DIR=C:/Users/polyn/AppData/Local/Temp/d20200817-1876-h06zk7&amp;&amp;  C:/Rub ..._x000D__x000A_</S><S S=\"Error\">+                                                                ~~_x000D__x000A_</S><S S=\"Error\">The token '&amp;&amp;' is not a valid statement separator in this version._x000D__x000A_</S><S S=\"Error\">    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException_x000D__x000A_</S><S S=\"Error\">    + FullyQualifiedErrorId : InvalidEndOfLine_x000D__x000A_</S><S S=\"Error\"> _x000D__x000A_</S></Objs>",
 exit_status=1>
```

- [ ] All specs in lib\plugins\inspec-plugin-manager-cli\test\functional\install_test.rb` have this same error expecting a stdout but receive:
```
PluginManagerCliInstall#test_install_from_path_when_path_refers_to_a_versioned_gem_install_missing_extension [C:/Users/polyn/inspec/inspec/lib/plugins/inspec-plugin-manager-cli/test/functional/install_test.rb:51]:
Expected "#< CLIXML\r\n<Objs Version=\"1.1.0.1\" xmlns=\"http://schemas.microsoft.com/powershell/2004/04\"><S S=\"Error\">At line:1 char:117_x000D__x000A_</S><S S=\"Error\">+ ... DIR=C:/Users/polyn/AppData/Local/Temp/d20200817-2796-lc4rc9&amp;&amp;  C:/Rub ..._x000D__x000A_</S><S S=\"Error\">+                                                                ~~_x000D__x000A_</S><S S=\"Error\">The token '&amp;&amp;' is not a valid statement separator in this version._x000D__x000A_</S><S S=\"Error\">    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException_x000D__x000A_</S><S S=\"Error\">    + FullyQualifiedErrorId : InvalidEndOfLine_x000D__x000A_</S><S S=\"Error\"> _x000D__x000A_</S></Objs>" to be empty.

```
Same as above with all `skip_windows!` in:

- [ ] `lib\plugins\inspec-plugin-manager-cli\test\functional\list_test.rb`
- [ ] `lib\plugins\inspec-plugin-manager-cli\test\functional\uninstall_test.rb`
- [ ] `lib\plugins\inspec-plugin-manager-cli\test\functional\update_test.rb`

It appears Powershell does *not* like &&, so we may need to find where these are used and have an allowance for windows: https://medium.com/@spwallace1016/a-quick-note-the-operator-doesn-t-work-in-powershell-platform-ide-terminal-in-atom-editor-d52996033cbb

( @clintoncwolfe we run the tests in lib/plugins/... in CI, right?):

**Comment by Schwad** [comment]
It appears the actual error is PowerShell doesn't like the double ampersand, so would be worth seeing if there's any affiliated issues so we can run this and pass our tests on windows. https://medium.com/@spwallace1016/a-quick-note-the-operator-doesn-t-work-in-powershell-platform-ide-terminal-in-atom-editor-d52996033cbb

**Comment by Schwad** [comment]
This appears to be fixed in #5213 so will make sure to close this when that merged 🎉
